fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -0,0 +1,22 @@
+This is an Android VNC server using contemporary Android 5+ APIs. It therefore does not require root access. In reverence to the venerable <a href='https://github.com/oNaiPs/droidVncServer'>droid-VNC-server</a> is is called droidVNC-NG.
```

Comment:
It would be better to auto-generate this from README.md or leave out the "Features" section.

Reply:
Done.